pom.xml

```diff
@@ -135,13 +135,14 @@
     <module>adapters</module>
     <module>application</module>
     <module>bom</module>
-    <module>config</module>
+    <module>demo-certs</module>
     <module>core</module>
     <module>client</module>
     <module>example</module>
     <module>legal</module>
     <module>server</module>
     <module>tests</module>
+    <module>dispatchrouter</module>
```

Comment:
You may have noted that the modules have been ordered alphabetically so far ... ;-)